index.bs

```diff
@@ -65,16 +65,19 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
+The device orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
 
-Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
-of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
+The device orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
+of the [=low-level=] motion sensors (accelerometer, gyroscope). The type of fusion (complementary
```

Comment:
> I guess that the advantage of the "rotation sensor" in comparison with "geomagnetic rotation sensor" is that the gravity vector is more stable. Ie. instead of isolating gravity from say a accelerometer using a low pass filter, you can get a more precise reading using the z component of a complementary filter between accelerometer and gyroscope.

Low pass filters aren't very precise plus they create delay (i.e. increase latency), which is why you want to avoid them.

Gyroscope are quite battery-hungry as they're actually oscillating at high speeds (> 20K Hz), but they provide nice smooth curves and low latency.

> The disadvantage is draining the battery quicker, for a bit of extra precision.

Well, low latency is really what's worth the battery cost, not extra precision.